### ISSUE TITLE
Presentation Feedback

### ISSUE THREAD

**Issue by dsnguon-1541830** [angellinn/us-manufacturing#5]
The data of your dashboard is highly detailed and covers your topic very well. The only thing that I would change is making the the state and manufacturing scales highly differing gradient colors because some people with color-blindness might find it difficult to read them.

**Comment by angellinn** [comment]
Yes I agree! I'll definitely make adjustment to that. Thank you for your feedback!
